### PR TITLE
Use organization level templates

### PR DESCRIPTION
**Pull request by Arcanemagus**
### Requirements for Contributing Documentation

* Fill out the template below. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* The pull request must only contribute documentation (for example, markdown files or API docs). To contribute other changes, you must use a different template. You can see all templates at https://github.com/atom/atom/tree/master/.github/PULL_REQUEST_TEMPLATE.

### Description of the Change

Remove the local templates for issues and pull requests in favor of the organization wide ones, meaning we only need to update those for changes. Currently these templates are essentially identical so no changes should be visible in the process.

The only change made that should be visible is that the legacy `ISSUE_TEMPLATE.md` has been removed, meaning this link will open a blank issue:
image

Since that is now a legacy functionality I don't think this is too big of a change, especially since we would automatically close such an issue anyway for failing to follow the template.

~~🚨 ⚠️ **Warning!!!** ⚠️ 🚨~~
~~The `bit.ly` links in [PULL_REQUEST_TEMPLATE.md](https://github.com/atom/atom/blob/master/PULL_REQUEST_TEMPLATE.md) need to be updated to point to the `atom/.github` locations before this is merged!~~

### Release Notes

N/A

-----
[View rendered .github/ISSUE_TEMPLATE/Feature_request.md](https://github.com/atom/atom/blob/la/organization-templates/.github/ISSUE_TEMPLATE/Feature_request.md)
[View rendered .github/ISSUE_TEMPLATE/bug_report.md](https://github.com/atom/atom/blob/la/organization-templates/.github/ISSUE_TEMPLATE/bug_report.md)
[View rendered .github/PULL_REQUEST_TEMPLATE/bug_fix.md](https://github.com/atom/atom/blob/la/organization-templates/.github/PULL_REQUEST_TEMPLATE/bug_fix.md)
[View rendered .github/PULL_REQUEST_TEMPLATE/documentation.md](https://github.com/atom/atom/blob/la/organization-templates/.github/PULL_REQUEST_TEMPLATE/documentation.md)
[View rendered .github/PULL_REQUEST_TEMPLATE/feature_change.md](https://github.com/atom/atom/blob/la/organization-templates/.github/PULL_REQUEST_TEMPLATE/feature_change.md)
[View rendered .github/PULL_REQUEST_TEMPLATE/performance_improvement.md](https://github.com/atom/atom/blob/la/organization-templates/.github/PULL_REQUEST_TEMPLATE/performance_improvement.md)
[View rendered CONTRIBUTING.md](https://github.com/atom/atom/blob/la/organization-templates/CONTRIBUTING.md)
[View rendered ISSUE_TEMPLATE.md](https://github.com/atom/atom/blob/la/organization-templates/ISSUE_TEMPLATE.md)
[View rendered PULL_REQUEST_TEMPLATE.md](https://github.com/atom/atom/blob/la/organization-templates/PULL_REQUEST_TEMPLATE.md)

### CODE REVIEW

**Comment by Arcanemagus** [comment]
@jasonrudolph It looks like you were the one who originally created those links, assuming this is a direction we want to go with can you update those to point to the organization level templates? Since those files already exist this can be done at any time.

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ⚛👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.
 
 1. Copy the correct template for your contribution
-  - 🐛 Are you fixing a bug? Copy the template from https://bit.ly/atom-bugfix
-  - 📈 Are you improving performance? Copy the template from https://bit.ly/atom-perf
-  - 📝 Are you updating documentation? Copy the template from https://bit.ly/atom-docs
-  - 💻 Are you changing functionality? Copy the template from https://bit.ly/atom-behavior
+  - 🐛 Are you fixing a bug? Copy the template from <https://bit.ly/atom-bugfix>
```
Comment (jasonrudolph) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - 🐛 Are you fixing a bug? Copy the template from <https://bit.ly/atom-bugfix-pr>
```

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+  - 📈 Are you improving performance? Copy the template from <https://bit.ly/atom-perf>
+  - 📝 Are you updating documentation? Copy the template from <https://bit.ly/atom-docs>
```
Comment (jasonrudolph) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - 📝 Are you updating documentation? Copy the template from <https://bit.ly/atom-docs-pr>
```

File: PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+  - 💻 Are you changing functionality? Copy the template from <https://bit.ly/atom-behavior>
```
Comment (jasonrudolph) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - 💻 Are you changing functionality? Copy the template from <https://bit.ly/atom-behavior-pr>
```

File: PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -1,10 +1,10 @@
 ⚛👋 Hello there! Welcome. Please follow the steps below to tell us about your contribution.
 
 1. Copy the correct template for your contribution
-  - 🐛 Are you fixing a bug? Copy the template from https://bit.ly/atom-bugfix
-  - 📈 Are you improving performance? Copy the template from https://bit.ly/atom-perf
-  - 📝 Are you updating documentation? Copy the template from https://bit.ly/atom-docs
-  - 💻 Are you changing functionality? Copy the template from https://bit.ly/atom-behavior
+  - 🐛 Are you fixing a bug? Copy the template from <https://bit.ly/atom-bugfix>
+  - 📈 Are you improving performance? Copy the template from <https://bit.ly/atom-perf>
```
Comment (jasonrudolph) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
  - 📈 Are you improving performance? Copy the template from <https://bit.ly/atom-perf-pr>
```

**Review by jasonrudolph — COMMENTED** [review]
> can you update those to point to the organization level templates?

@Arcanemagus: As far as I can tell, you can't change the target URL for a bit.ly link, so I think we'll have to create _new_ bit.ly links. I've included some suggested edits below to point to new bit.ly links for the pull request templates in the organization-level repo.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -163,7 +163,7 @@ Include details about your configuration and environment:
...
 
-Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](ISSUE_TEMPLATE.md), including the steps that you imagine you would take if the feature you're requesting existed.
+Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that you don't need to create one. When you are creating an enhancement suggestion, please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill in [the template](https://github.com/atom/.github/blob/master/.github/ISSUE_TEMPLATE/bug_report.md), including the steps that you imagine you would take if the feature you're requesting existed.
```
[the remark this answers is not in the document]
Comment (Arcanemagus) [reply CONTRIBUTING.md]: Ha! I just was fixing the links, didn't actually check the content but you are absolutely right.

**Comment by Arcanemagus** [comment]
Confirmed that issues are now using the `atom/.github` templates after merging, the "Open a regular issue" link behaves as explained, and PR's have the new links in the template message inserted.

**Pull request merged**